Server.cs

```diff
@@ -99,29 +100,29 @@ public class Server : Node
 
   void SendGameUpdates()
   {
-    _serilogger.Verbose("Server.cs: Sending updates about game state to clients");
-
-    Godot.Collections.Array players = _GetNodesFromGroup("player_ships");
-    foreach (PlayerShip player in players)
-    {
-      _serilogger.Verbose($"Server.cs: Sending update for player: {player.uuid}");
-      // create the buffer for the specific player
-      GameEvent gameEvent = player.CreatePlayerGameEventBuffer(GameEvent.GameEventType.GameEventTypeUpdate);
-
-      // send the event for the player
-      MessageInterface.SendGameEvent(gameEvent);
-    }
-
-    // TODO: we never send a create message for the missile
-    foreach (SpaceMissile missile in GetTree().GetNodesInGroup("missiles"))
-    {
-      _serilogger.Verbose($"Server.cs: Processing missile: {missile.uuid}");
-      // create the buffer for the missile
-      GameEvent gameEvent = missile.CreateMissileGameEventBuffer(GameEvent.GameEventType.GameEventTypeUpdate, missile.MyPlayer.uuid);
-
-      // send the buffer for the missile
-      MessageInterface.SendGameEvent(gameEvent);
-    }
+	_serilogger.Verbose("Server.cs: Sending updates about game state to clients");
```

Comment:
looks like a lot of changes in indentation (2space versus 2tab, etc). We should pick a standard and stick to it probably. I'm not passionate either way. I think Godot's standard is tabs with two spaces per tab.

Reply:
Went back and took a look at what was going on with the spurious spacing changes and went through each file to ensure spaces instead of tabs as that's what was being used in the tip of main.